grow/pods/routes.py

```diff
@@ -186,6 +186,7 @@ def match_error(self, path, status=404):
 
     def reset_cache(self, rebuild=True, inject=False):
         if rebuild:
+            self.pod.podcache.reset()
             self._build_routing_map(inject=False)
```

Comment:
Sounds like a good idea. I think with the improvements to the cache file it may not be an issue. I'm going to do some testing next to see what change to the route loading it makes to have the cache file. So far the changes don't actually use the cache file.